Sources/FluentKit/Properties/Field.swift

```diff
@@ -26,14 +26,14 @@ public final class ModelField<Model, Value>: AnyField, FieldRepresentable
                 case .bind(let bind):
                     return bind as! Value
                 case .default:
-                    fatalError("Cannot access default field before it is initialized or fetched")
+                    fatalError("Cannot access default field for '\(key)' before it is initialized or fetched")
```

Comment:
Now that `FieldProperty` includes the base model, it could be nice to include that information here to make it even easier to debug: `\(Model.self).\(key)`

Reply:
There were no conflicts when Caleb approved it :P